train.py

```diff
@@ -75,9 +75,17 @@ def train(
         description="Whether to run validation during training.", 
         default=True
     ),
+    validation_prompt: str = Input(
+        description="Prompt to use for generation during validation. If provided, a response to this prompt will be sampled and logged during validation.",
+        default=None,
+    ),
     learning_rate: float = Input(
         description="learning rate, for learning!", default=1e-4, ge=0
     ),
+    wrap_packed_sequences: bool = Input(
+        description="This is hard to describe succinctly. Maybe we don't need to expose this.",
```

Comment:
yeah it's an interesting point, this is hard to describe. I almost wonder (or rather, I do wonder but I don't know for sure) if it makes sense to instead just expose a `pack sequences` flag, and then for `prompt:` `completion:` datasets we don't wrap but for `text:` datasets we do (i.e. users don't get control of this).

...as I think about this, it probably makes the most sense to expose this for now while we run experiments, and then we can re-evaluate once we have our very good excellent killer fine-tunes.

Reply:
Agreed! I exposed control over packing vs no packing and chunk size, and we can revisit later.